jaeger_client/config.py

```diff
@@ -319,8 +321,14 @@ def new_tracer(self, io_loop=None):
                 max_operations=self.max_operations)
         logger.info('Using sampler %s', sampler)
 
+        sender = UDPSender(
+            host=self.local_agent_reporting_host,
+            port=self.local_agent_reporting_port,
+            io_loop=io_loop
+        )
+
         reporter = Reporter(
-            channel=channel,
+            sender=sender,
```

Comment:
@black-adder renaming the field is a breaking change. Are you sure this can be merged?

Reply:
doh reverting

Reply:
let's find a way to let this PR to get merged, but in backwards compatible way.

Reply:
I did changes to make it backwards compatible. Should I open another PR?